08-Secure by default/Secure Templates/ARM/ServiceBus.json

```diff
@@ -26,11 +26,13 @@
   },
   "resources": [
     {
-      "apiVersion": "2015-08-01",
+      "apiVersion": "2022-01-01-preview",
       "name": "[parameters('serviceBusNamespaceName')]",
       "type": "Microsoft.ServiceBus/namespaces",
       "location": "[resourceGroup().location]",
-      "properties": {},
+      "properties": {
+        "minimumTlsVersion": "1.2"
```

Comment:
Add comment to map that this property is required for control 'Azure_ServiceBus_XYZ'

Reply:
Addressed.